auth/user_mgt.go

```diff
@@ -66,6 +69,7 @@ type multiFactorInfoResponse struct {
 }
 
 // MultiFactorInfo describes a user enrolled second phone factor.
+// TODO : convert PhoneNumber to PhoneMultiFactorInfo struct
```

Comment:
Is this still a TODO, I wonder if this will show up in the reference docs for `MultiFactorInfo` type.

Reply:
This is a TODO for later stages when we want to add more MFA methods such as TOTP, etc. If it does show up in the reference docs, we can report this as a bug and remove from here.